cmd/kueueviz/backend/README.md

```diff
@@ -63,3 +63,9 @@ KUEUE_VIZ_PORT=8181 ./kueue_ws_app
 | `/ws/cohort/{cohort_name}`                        | Streams updates for a specific cohort   |
 | `/ws/workload/{namespace}/{workload_name}`        | Streams updates for a specific workload |
 | `/ws/workload/{namespace}/{workload_name}/events` | Streams events for a specific workload  |
+
+### REST API
+
+| Endpoint                                          | Description                             |
+| ------------------------------------------------- | --------------------------------------- |
+| `/api/yaml/{resourceType}/{name}?namespace={ns}` | Returns YAML content for a specific resource |
```

Comment:
Maybe like this?

```suggestion
| `/api/{resource_type}/{namespace}/{name}?output={output_type}` | Returns content for a specific resource |
```

WDYT?

Reply:
+1, especially as we may introduce similar API for showing `describe` (similar to k9s), or `json` outputs.

Reply:
This is a good idea, I will optimize it later.

Reply:
Explain why REST.

Initially, my design was based on websocket. But, I considered that viewing YAML is more of a one-time fetch action, and the YAML content might be large. Maybe REST is more appropriate to handle this scenario.

as @mimowo say, I'm indeed considering adding more capabilities like View JSON and etc in the future. (By the way, k9s is great.)

And I want to ask how to consider adding create/update capabilities in the future.